CODE_OF_CONDUCT.md

```diff
@@ -23,6 +23,7 @@ Examples of unacceptable behavior by participants include:
 - Public or private harassment
 - Publishing others’ private information, such as a physical or electronic address, without explicit
   permission
+- Refusing to accept decisions or reasonable requests of project maintainers
```

Comment:
I think we should make it "arguing with maintainers after being asked to stop".

Reply:
I figured this covers both the "we have decided to design it this way" and "plz stop"